src/components/Thought.js

```diff
@@ -372,6 +382,9 @@ const ThoughtContainer = ({
   distance,
   dragPreview,
   dragSource,
+  dragInProgress = false,
+  draggedThoughtsRanked,
+  dragHold,
```

Comment:
Yeah, you're right, thanks. I moved highlighting logic to `BulletCursorOverlay` component. `isDragging` from react-dnd monitor is still passed to `ThoughtComponent`, so it will cause re-render on dragStart. Other part of logic are in new component now. It works, but i'm a little confused in css rules in App.css. I tried to add the same rules as to  `.dragging > .... > .bullet-cursor-overlay` selectors, but it didn't work. Could you point me out please, what rules use for it?

Reply:
Cool, that does look better.

I don't quite understand your question. Are you asking where something is, or how to solve something?